deaddrop/settings.py

```diff
@@ -37,9 +37,11 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    
     "backend.apps.BackendConfig",
     "rest_framework",
-    # "backend",
+    "corsheaders",
+    # "backend", # why does this not work? idk. think it's bc of the above

```

Comment:
`backend` is not a registerable app on its own afaik, was there some resource that told you to add it?